src/python/era_5g_object_detection_common/era_5g_object_detection_common/face_detector.py

```diff
@@ -2,12 +2,16 @@
 from abc import ABC
 
 import cv2
+import numpy as np
+from typing import List, Tuple
 
 from era_5g_object_detection_common.image_detector import ImageDetector, ImageDetectorInitializationFailed
 
 # path to the face detector model file (the haarcascade xml)
 MODEL_FILE = os.getenv("NETAPP_FACE_DETECTOR_MODEL_FILE", None)
 
+FaceDetectorResultType = List[Tuple[List[float], float, int, str]]
```

Comment:
Sorry for another comment... :-D But shouldn't be this type defined in `era_5g_object_detection_common.image_detector` and imported where needed? Now the same thing is defined twice.

Reply:
Yes, we can do that, but we need to be a bit more careful then, since this type is mainly used for the old basic detectors and the newer ones can return different results (for example depending on whether the there are detections with masks as well). I moved it and renamed it to BasicDetectorResultType. Would you consider it fine like this?